drivers/flash/Kconfig.mcux

```diff
@@ -34,10 +34,17 @@ config FLASH_MCUX_FLEXSPI_NOR
 	select MEMC
 	select MEMC_MCUX_FLEXSPI
 
+config FLASH_MCUX_FLEXSPI_MX25UM51345G
+	bool "MCUX FlexSPI NOR driver"
```

Comment:
Update this string to "MCUX FlexSPI MX25UM51345G driver"

Reply:
Done.